extensions/sd_api_pictures/script.py

```diff
@@ -265,6 +268,27 @@ def custom_css():
     path_to_css = Path(__file__).parent.resolve() / 'style.css'
     return open(path_to_css, 'r').read()
 
+def get_checkpoints():
+    global params
+
+    models = requests.get(url=f'{params["address"]}/sdapi/v1/sd-models')
+    options = requests.get(url=f'{params["address"]}/sdapi/v1/options')
+    options_json = options.json()
+    params['sd_checkpoint'] = options_json['sd_model_checkpoint']
+    list = models.json()
+    params['checkpoint_list'] = [0] * len(list)
+    i = 0
+    for result in list:
+        params['checkpoint_list'][i] = result['title']
+        i = i + 1
```

Comment:
```suggestion
    params['checkpoint_list'] = [result["title"] for result in models.json()]
```
This can be simplfied.